proto/teleport/lib/teleterm/vnet/v1/vnet_service.proto

```diff
@@ -80,4 +80,5 @@ enum BackgroundItemStatus {
   BACKGROUND_ITEM_STATUS_ENABLED = 2;
   BACKGROUND_ITEM_STATUS_REQUIRES_APPROVAL = 3;
   BACKGROUND_ITEM_STATUS_NOT_FOUND = 4;
+  BACKGROUND_ITEM_STATUS_NOT_SUPPORTED = 5;
```

Comment:
Negative values for enum are [supported but not recommended](https://protobuf.dev/programming-guides/proto3/#enum), so I just used `5` instead. On top of that, `BackgroundItemStatus` in protos already doesn't map 1:1 to [the actual status returned by macOS](https://github.com/gravitational/teleport/blob/e552c01539b2f42e535f5bae84af6ba12aebd7f2/lib/vnet/daemon/client_darwin.go#L195-L203), because `0` returned by macOS means "not registered", whereas in protos 0 should be reserved "unspecified".